java/src/main/java/Client.java

```diff
@@ -0,0 +1,66 @@
+
+import com.azure.identity.ClientSecretCredential;
+import com.azure.identity.ClientSecretCredentialBuilder;
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusErrorContext;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessageContext;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+public class Client {
+
+    public void start() {
+
+        var credential = createCredential();
+
+        var processor = new ServiceBusClientBuilder()
+                .fullyQualifiedNamespace(Settings.getNameSpace())
+                .credential(credential)
+                .processor()
+                .queueName(Settings.getQueueName())
+                .processMessage(this::processMessage)
+                .processError(this::processError)
+                .buildProcessorClient();
+
+        processor.start();
+
+    }
+
+    private ClientSecretCredential createCredential() {
+        return new ClientSecretCredentialBuilder()
+                .tenantId(Settings.getTenantId())
+                .clientId(Settings.getClientId())
+                .clientSecret(Settings.getSecret())
+                .build();
+    }
+
+    private void processMessage(ServiceBusReceivedMessageContext context) {
+        writeMessage(context.getMessage().getBody().toString());
+    }
+
+    private void processError(ServiceBusErrorContext context) {
+        writeMessage(context.getException().getMessage());
```

Comment:
Could we throw an error or write an error to the console rather than writing the error to the output file? This would be more consistent with e.g. the Node client: https://github.com/elexon-data/iris-clients/blob/main/nodeJs/processors/processError.js

Reply:
Yes, done